dev/src/__tests__/navbarResponsive.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { Builder, By, Browser, IRectangle, Capabilities } from 'selenium-webdriver'
+import { isOverlapping } from './testHelpers'
+
+describe("Front page elements login and logo should not overlap", () => {
+    //Laptop, desktop, mobile
+    it("Should not overlap on res 1920x1080", async () => {
+        const firefox = require('selenium-webdriver/firefox')
+        const service = new firefox.ServiceBuilder('drivers\\geckodriver.exe')
+        const driver = new Builder().forBrowser('firefox').setFirefoxService(service).withCapabilities(Capabilities.firefox()
+            .set("acceptInsecureCerts", true)).build()
+        await driver.get("https://127.0.0.1/dev")
```

Comment:
Can we reuse the same driver for all these tests, rather than creating new ones for each individual test?